### ISSUE TITLE
Re-use of inferred parent names in large families

### ISSUE THREAD

**Issue by Awtum** [brentp/somalier#119]
In cases of large families (in this case merged from many smaller families during infer), inferred parents between sets of siblings appear to be re-used. This gives the appearance of many children of the same parents, when in reality it is multiple sets of siblings consisting of 2 or 3 each that may be distantly related.

**Comment by brentp** [comment]
Yes, `--infer` tries to do simple things to do the best it can, but in low-quality or low-coverage samples, it can make many mistakes.
